backend/tests/v5/e2e/routes/users.test.js

```diff
@@ -524,6 +530,11 @@ const testForgotPassword = () => {
 			expect(res.body.code).toEqual(templates.invalidArguments.code);
 		});
 
+		test('should not send email but return ok if user is an SSO user', async () => {
```

Comment:
if you're really doing what this test says is doing, you should be mocking the mailer to make sure it's not called 😆 

As it stands it only tests if it returns ok if it's a SSO user